CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ Changelog
 ### New Cisco Resources
 
 ### Added
+* Extend nxapi client for https support
+   * `use_ssl` will be true when `transport` is `https`
+   * `verify_mode` controls the OpenSSL session verification,
+      - 'none' for `OpenSSL::SSL::VERIFY_NONE`
+      - 'peer' for `OpenSSL::SSL::VERIFY_PEER`
+      - 'client-once' for `OpenSSL::SSL::VERIFY_CLIENT_ONCE`
```

Comment:
@Thomas-Franklin Did have a chance to test/verify these different modes?  I only used `OpenSSL::SSL::VERIFY_NONE`.   We need to make sure and test all of the modes before claming support for each.

Reply:
Unfortunately I have not been able too, will try and do further testing tomorrow and see if I can test the alternative modes and add the minitests suggested as part of this PR